tip-wlan/charts/wlan-port-forwarding-gateway-service/files/README.txt

```diff
@@ -0,0 +1,3 @@
+#This README prevents 
```

Comment:
I don't think we need this file or the folder, if it's not being used in the charts. We should update the configMap to remove reference to the `files` folder.